pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.apache.sling</groupId>
         <artifactId>sling-bundle-parent</artifactId>
-        <version>41</version>
+        <version>52</version>
```

Comment:
Please also add property `project.build.outputTimestamp` for reproducible builds (https://maven.apache.org/guides/mini/guide-reproducible-builds.html#how-do-i-configure-my-maven-build).

Reply:
Updated in [1bfd1cf](https://github.com/apache/sling-org-apache-sling-serviceusermapper/pull/9/commits/1bfd1cfbfaa4869419d5b0d98212d6b7a3908f09)